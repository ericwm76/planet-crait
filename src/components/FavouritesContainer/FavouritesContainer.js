import React from 'react';
import Favourite from '../Favourite/Favourite';

const FavouritesContainer = (props) => {

    return (
      <section className='Favourites-container'>
        <Favourite />
      </section>
    )
}

export default FavouritesContainer;