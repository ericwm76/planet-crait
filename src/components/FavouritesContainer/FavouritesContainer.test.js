import React from 'react';
import { shallow } from 'enzyme';
import FavouritesContainer from './FavouritesContainer';

describe('FavouritesContainer', () => {
  let wrapper;

  beforeEach(() => {
    wrapper = shallow(<FavouritesContainer />)
  })
  
  it('should match snapshot with all data passing in correctly', () => {
    expect(wrapper).toMatchSnapshot();
  })
})