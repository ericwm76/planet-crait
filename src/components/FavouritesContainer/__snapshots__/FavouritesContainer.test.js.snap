// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`FavouritesContainer should match snapshot with all data passing in correctly 1`] = `undefined`;
